$OpenBSD: patch-Rakefile,v 1.5 2021/01/19 19:01:50 rsadowski Exp $

`configure' defines LLVM_LLD, that can only be overridden with another path.
Ensure that the default and working linker is used on all clang archs.

Index: Rakefile
--- Rakefile.orig
+++ Rakefile
@@ -166,7 +166,6 @@ def setup_globals
   ldflags                  = ""
   ldflags                 += determine_stack_protector_flags
   ldflags                 += " -pg"                                     if c?(:USE_PROFILING)
-  ldflags                 += " -fuse-ld=lld"                            if is_clang? && !c(:LLVM_LLD).empty?
   ldflags                 += " -Llib/libebml/src -Llib/libmatroska/src" if c?(:EBML_MATROSKA_INTERNAL)
   ldflags                 += " -Llib/fmt/src"                           if c?(:FMT_INTERNAL)
   ldflags                 += " #{c(:EXTRA_LDFLAGS)} #{c(:USER_LDFLAGS)} #{c(:LDFLAGS_RPATHS)} #{c(:BOOST_LDFLAGS)}"
